src/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.csproj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$(RepoRoot)eng/pkg/Pack.props" />
   <PropertyGroup>
-    <Authors>Intel</Authors>
```

Comment:
><Authors>Intel</Authors> [](http://example.com/codeflow?start=4&length=24)

I assume this is ok and no concern removing it.

Reply:
It was wrong for us to claim Intel is the Author of the package.  The gallery doesn't even present it to our users.
https://www.nuget.org/packages/Microsoft.ML.Mkl.Redist/2.0.0-preview.22313.1
<img width="223" alt="image" src="https://user-images.githubusercontent.com/8918108/198308298-d78c4b80-25c0-4122-900c-52fe337aa69d.png">
We are the ones who build this package and binary, it's a [custom linked](https://github.com/dotnet/machinelearning/blob/04dda55ab0902982b16309c8e151f13a53e9366d/docs/building/MlNetMklDeps/README.md) copy of MKL.  The package itself has the Intel license.